### PR TITLE
Make Section ES Indexable

### PR DESCRIPTION
**Pull request by mparent61**
@collin @benghaziboy 

This will not parse/index the somewhat-arbitrary "query" JSON content -- would we ever need this?

### CODE REVIEW

**Comment by collin** [comment]
I'm building my env with django-bulbs pointed at branch out now to confirm it does what I need.

**Comment by benghaziboy** [comment]
should be good. Might want to add a test to confirm mapping, but we can go through that later.
:+1:

**Comment by vforgione** [comment]
:+1:

**Comment by collin** [comment]
@mparent61 @benghaziboy @vforgione Mapped the slug. Made it possible to facet on sections.

File: bulbs/sections/models.py [hunk 1]
```diff
@@ -26,6 +28,9 @@ class Section(models.Model):
 
     class Mapping:
         name = field.String(analyzer="autocomplete", fields={"raw": field.String(index="not_analyzed")})
+        slug = field.String(index="not_analyzed")
```
Comment (collin) [inline bulbs/sections/models.py]: slug map

**Pull request merged**